README.md

```diff
@@ -45,7 +45,7 @@ yarn add @storybook/test-runner -D
 Jest is a peer dependency. If you don't have it, also install it
 
 ```jsx
-yarn add jest -D
+yarn add jest@27.5.1 -D
```

Comment:
Any reason for that particular version rather than just @27?

Reply:
No reason in particular, I've used the version that I had in a repository I've set up for testing

Reply:
@yannbf removed the specific version with the latest commit. Let me know if you need anything else from me.